middleware/validator/users.js

```diff
@@ -6,18 +6,17 @@ const { User } = models;
  * Function validates user Signup inputs with customized error messages
  * @param {Object} req the request body
  * @param {Object} res the response body
- * @param {function} next a call to the nect function
+ * @param {function} next a call to the next function
  * @returns {undefined}
- */
-export default function signup(req, res, next) {
+ */const signup = (req, res, next) => {
```

Comment:
There should be a new line after the block comment or before the  `signup` function.